src/jscpd.coffee

```diff
@@ -32,26 +33,21 @@ class jscpd
       debug: off
       files: null
       exclude: null
-    , options
+    , config
 
-    optionsNew = _.extend optionsNew, config
     for key, value of options
-      if value is not null then optionsNew[key] = value
```

Comment:
This is a bug, I believe. 

Compiles to
```
if (value === !null) {
  optionsNew[key] = value;
}
```

I don't think this is the intention.